packages/core/src/schema/types.ts

```diff
@@ -58,7 +58,7 @@ export type LlmAsJudgeEvaluationMetadata = InferSelectModel<
 export type Subscription = InferSelectModel<typeof subscriptions>
 
 export type EvaluationDto = Evaluation & {
-  configuration?: LlmAsJudgeEvaluationMetadata['configuration']
+  configuration: LlmAsJudgeEvaluationMetadata['configuration'] | null
```

Comment:
why the null? isn't it supposed to be not null after your migration ☝🏼 ?

Reply:
This is the original `configuration` attribute from the `evaluations`, not the new one in the `LegacyMetadata`, so technically every value should be Null here now. This is just for compatibility while I'm doing changes and will be removed in the future.